apiary.apib

```diff
@@ -83,24 +83,139 @@ TODO
 
 
 # Group bootstrap
+Show or edit the list of bootstrap peers.
+
+Running 'ipfs bootstrap' with no arguments will run 'ipfs bootstrap list'.
+
+SECURITY WARNING:
+
+The bootstrap command manipulates the "bootstrap list", which contains
+the addresses of bootstrap nodes. These are the *trusted peers* from
+which to learn about other peers in the network. Only edit this list
+if you understand the risks of adding or removing nodes from this list.
 
 ## add [GET /bootstrap/add]
+Add peers to the bootstrap list
+
+Outputs a list of peers that were added (that weren't already
+in the bootstrap list).
+
++ Parameters
+    + arg (Multiaddr, required) - A peer to add to the bootstrap list
+    + default (boolean, optional) - add default bootstrap nodes
 
 + Response 200 (text/json)
 
+    + Attributes (object)
+        + Peers (array)
+            - (Multihash)
+
+    + Body
+
+        ```
+        {
+          "Peers": [
+            "/ip4/5.9.33.222/tcp/4001/ipfs/QmNRV7kyUxYaQ4KzxFXPYm8EfuzJbtGn1wSFenjXL6LD8y"
+          ]
+        }
+        ```
+
++ Response 400 (text/json)
```

Comment:
Changed, but I'm not sure how to tell the difference. Is `text/plain` also a valid mimetype for some responses?